File: KarateKlub/KarateKlub.Api/Models/TakmicariPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class TakmicariPregledVM
    {
        public class Row {
            public int id;
            public string takmicar;
            public string regBroj;
            public byte[] slika;

        }

        public List<Row> rows;


    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/TakmicariPregledVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.List;

public class TakmicariPregledVM implements Serializable {
    public static class Row implements Serializable{
        public int id;
        public String takmicar;
        public String regBroj;
        public String slika;
    }
    public List<Row> rows;
}
